db/migrate/20171012163307_add_input_fields_to_database.rb

```diff
@@ -0,0 +1,17 @@
+class AddInputFieldsToDatabase < ActiveRecord::Migration[5.1]
+  def change
+    add_column :medicaid_applications, :first_name, :string
+    add_column :medicaid_applications, :last_name, :string
+    add_column :medicaid_applications, :gender, :string
+
+    add_column :medicaid_applications, :income_unemployment, :boolean
+    add_column :medicaid_applications, :income_pension, :boolean
+    add_column :medicaid_applications, :income_social_security, :boolean
+    add_column :medicaid_applications, :income_retirement, :boolean
+    add_column :medicaid_applications, :income_alimony, :boolean
```

Comment:
You guys probably already looked at making these a `jsonb` datatype with a hash in it, right? I'm assuming there's some good reason not to go that way...

Reply:
here I am just copying the way we did it for the SNAP flow. I think the benefit of having separate columns is it's easier to render the correct form state when re-visiting a page (rails does it for us ratherthan having to fill in fields based on a hash of data)